packages/esm-patient-flags-app/src/index.ts

```diff
@@ -0,0 +1,45 @@
+import { defineConfigSchema, getAsyncLifecycle } from '@openmrs/esm-framework';
+import { configSchema } from './config-schema';
+
+const importTranslation = require.context('../translations', false, /.json$/, 'lazy');
+
+const backendDependencies = {
+  kenyaemr: '^18.2.0',
```

Comment:
So, policy note: we can't be depending on kenyaemr here (i.e., if this depends on KenyaEMR, it is no longer a good candidate for the RefApp and hence for this repository). If they patientflags feature is implemented in the KenyaEMR module than it needs to be extracted into it's own thing... However, there's already a patientflags module and I suspect that's what should be referenced here instead.